packages/edit-post/src/plugins/copy-content-menu-item/index.js

```diff
@@ -1,34 +1,32 @@
 /**
  * WordPress dependencies
  */
-import { ClipboardButton } from '@wordpress/components';
+import { MenuItem } from '@wordpress/components';
 import { withDispatch, withSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
-import { withState, compose } from '@wordpress/compose';
+import { useCopyOnClick, compose } from '@wordpress/compose';
+import { useRef, useEffect } from '@wordpress/element';
+
+function CopyContentMenuItem( { createNotice, editedPostContent } ) {
+	const ref = useRef();
```

Comment:
`copyRef` will make it easier to read. Otherwise it’s a beautiful piece of magic 😃

Reply:
Yeah it's hard to see that the clipboard package is handling the click behaviour. Imo the name is a bit misleading. It pretty much only handles copying something to the clipboard on click.